packages/ui-text-area/src/TextArea/index.tsx

```diff
@@ -318,31 +282,29 @@ class TextArea extends Component<TextAreaProps> {
         defaultValue={defaultValue}
         placeholder={placeholder}
         ref={(textarea, ...args) => {
-          // @ts-expect-error ts-migrate(2339) FIXME: Property '_textarea' does not exist on type 'TextA... Remove this comment to see the full error message
           this._textarea = textarea
-          // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-          textareaRef.apply(this, [textarea].concat(args))
+          if (typeof textareaRef === 'function') {
+            textareaRef.apply(this, [textarea, ...args])
+          }
         }}
-        // @ts-expect-error ts-migrate(2322) FIXME: Type '{ width: string | undefined; resize: "none" ... Remove this comment to see the full error message
         style={style}
         id={this.id}
         required={required}
         aria-required={required}
-        aria-invalid={this.invalid ? 'true' : null}
+        aria-invalid={this.invalid ? 'true' : 'false'}
         disabled={disabled || readOnly}
         css={this.props.styles?.textArea}
         onChange={this.handleChange}
       />
     )
 
     return (
-      // @ts-expect-error ts-migrate(2769) FIXME: No overload matches this call.
       <FormField
+        label={null}
         {...pickProps(this.props, FormField.allowedProps)}
```

Comment:
I bumped into this before too. TS doesn't know that pickProps will give back the `label` props, so I just put it there explicitly:

```js
<FormField
  {...pickProps(this.props, FormField.allowedProps)}
  label={this.props.label}
```

Reply:
Nice idea, did not think of that